<?xml version="1.0" encoding="UTF-8"?>
<launch>

 <!-- Formation params file -->
  <!--<arg name="FORMATION_PARAMS_FILE" default="$(find shared_autonomy)/config/formation_params.yaml" />
  <rosparam file="$(arg FORMATION_PARAMS_FILE)" command="load"/>-->

 <!-- Optitrack parameters -->
 <arg name="server" default="192.168.2.50"/> <!-- for connecting via wifi to turtle router -->

  <node pkg="vrpn_client_ros" type="vrpn_client_node" name="vrpn_client_node" output="screen">
    <rosparam subst_value="true">
      server: $(arg server)
      port: 3883

      update_frequency: 100.0
      frame_id: world

      # Use the VRPN server's time, or the client's ROS time.
      use_server_time: false
      broadcast_tf: true

      # Must either specify refresh frequency > 0.0, or a list of trackers to create
      refresh_tracker_frequency: 1.0
      #trackers:
      #- FirstTracker
      #- SecondTracker
    </rosparam>
  </node>

</launch>
